bibliopixel/animation/animation.py

```diff
@@ -119,7 +119,8 @@ def run_context(self):
 
         self.check_delay()
 
-        self.preRun(self.runner.amt)
+        self.layout.all_off()
```

Comment:
Yeah... I agree. The reason I did this is a lot of animations relied on the default preRun calling all_off, but I wanted to move that out of preRun so that they didn't have to always still call all_off if they overrode preRun. My intent with all this preRun stuff is that I ran into issues with the remote because it loads all of the animation classes at start. This meant that if one of them did something like access the webcam it did so from the beginning, even if the animation was not running. I had two options... 
- Late load all animations on run request. I didn't like this because I'd rather have an upfront load delay then clicking the button and waiting.
- Require certain initialization to happen in preRun instead.
I went with the latter and for another good reason. If you do it that way and also implement cleanup() on your animation (if you need it), then starting the animation from the UI *always* starts the animation back at the beginning state. I noticed that this was not the case... on ScrollText I could keep tapping the button for the animation and it would start where it left off. Not what I wanted.
So, yeah... the plan no is that if you need a specific starting state it MUST go in prerun (assuming you want to support the remote... script runners won't see a diff).